Makefile

```diff
@@ -73,6 +77,22 @@ kube-clean:
 	kubectl delete rc deis-${SHORT_NAME}
 	kubectl delete svc deis-${SHORT_NAME}
 
+kube-delete:
```

Comment:
`kube-delete` and `kube-create` seem to have a lot of overlapping functionality with `kube-clean`, `kube-rc`/`kube-service` (respectively). Can you delete one (don't have any preference which) or merge them?